packages/core-components/src/components/Link/Link.tsx

```diff
@@ -46,12 +47,40 @@ const useStyles = makeStyles(
 
 export const isExternalUri = (uri: string) => /^([a-z+.-]+):/.test(uri);
 
+/**
+ * Get the app base path from the configured app baseUrl.
+ *
+ * The returned path does not have a trailing slash.
+ */
+const getBasePath = (configApi: Config) => {
+  let { pathname } = new URL(
+    configApi.getOptionalString('app.baseUrl') ?? '/',
+    'http://dummy.dev', // baseUrl can be specified as just a path
+  );
+  pathname = pathname.replace(/\/*$/, '');
+  return pathname;
+};
+
 export type LinkProps = MaterialLinkProps &
   RouterLinkProps & {
     component?: ElementType<any>;
     noTrack?: boolean;
   };
 
+export const useResolvedPath = (uri: LinkProps['to']) => {
+  const configApi = useApi(configApiRef);
```

Comment:
Ah-hah.  Looks like the failed tests are coming from _other_ plugins that use `<Link />` that don't wrap their tests in the app or a mocked config API.

Is it possible to make this hook use the config API optionally?

Reply:
Esse erro seria esperado mesmo ao usar `getOptionalString`?

Reply:
I'm asking because we're not mocking configApi in Link's old tests and they're not failing 🤔 ...

Reply:
Maybe set basePath to empty string... I'll try!

Reply:
Done!